daml-lf/transaction/src/main/protobuf/com/daml/lf/value.proto

```diff
@@ -18,7 +18,7 @@ import "google/protobuf/empty.proto";
 
 message VersionedValue {
     string version = 1;
-    Value value = 2;
+    bytes value = 2;
```

Comment:
Have you considered using an alternative such as [Any](https://developers.google.com/protocol-buffers/docs/proto3#any)? This change is not recommended as we lose type safety.